Makefile

```diff
@@ -469,46 +502,46 @@ frwikisource_tuning_reports: \
 	tuning_reports/frwikisource.page_level.md
 
 ########################## Russian Wikipedia ###################################
-datasets/ruwiki.labelings.20160501.json:
+datasets/ruwiki.labelings.20181201.json:
```

Comment:
"labeled_revisions"?

Reply:
A "labeling" is when a label is applied.  A "labeling_revision" is the most recent revision of an article at the time that a label was applied (to the talk page).  It's confusing and "labeled revision" would also be an appropriate term since ultimately, we are associating a label with a revision.